### DISCUSSION TITLE
Show text in the middle

### DISCUSSION THREAD

**Discussion by Toolmaster30** [Clooos/Bubble-Card#708] [General]
Good morning, I've been trying to center the text of the bubble card for days. Is that possible somehow?

Greetings, Georg

**Comment by Mystically11** [comment]
Yes, using custom css

in the UI editor, paste this in the dropdown called "Custom Styles / Templates" that's located inside the "Styling options" dropdown:
```
.bubble-state, .bubble-name {
  justify-content: center !Important;
}
```

**Comment by Toolmaster30** [comment]
Thanks. It only works if the text scrolling effect is activated.

**Reply by Mystically11** [reply]
try adding `text-align: center !Important;` in there

**Comment by Toolmaster30** [comment]
Thanks.
